# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: architecture/avr32/package/*/hotfix-texinfo.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- gdb-6.7.1.atmel.1.0.3/bfd/doc/elf.texi.vanilla	2021-10-11 12:08:24.192828941 +0200
+++ gdb-6.7.1.atmel.1.0.3/bfd/doc/elf.texi	2021-10-11 12:08:38.037828116 +0200
@@ -8,7 +8,7 @@
 haven't bothered yet.
 
 @findex bfd_elf_find_section
-@subsubsection @code{bfd_elf_find_section}
+@subsection @code{bfd_elf_find_section}
 @strong{Synopsis}
 @example
 struct elf_internal_shdr *bfd_elf_find_section (bfd *abfd, char *name);
